src/dctest/core.cljs

```diff
@@ -51,30 +52,42 @@ Options:
                            (.inspect exec exec-cb)))
         (@check-fn)))))
 
+(defn ident-output [s]
```

Comment:
FYI there's an indent (from conlink) in dctest.util.  (Not identical, but I think would work here?)